Sources/UtilityBeltNetworking/HTTPClient+Combine.swift

```diff
@@ -5,9 +5,19 @@ import Foundation
 
 @available(iOS 13, macOS 10.15, tvOS 13.0, watchOS 6, *)
 extension HTTPClient {
+    // MARK: Data Response
+    
+    /// Creates a request publisher which fetches raw data from an endpoint.
+    /// Returns a `URLSessionTask`, which allows for cancellation and retries.
```

Comment:
Should this say that it returns a Publisher instead of a URLSessionTask? Or it could possibly be removed as the first line says the method creates a publisher and the return documentation also states this.

Reply:
Yep, should be removed, good call. Was doing a bunch of doc cleanup and added `- Returns` in the other area, I guess it was missing because of this line, strangely enough. Nice catch!